### PR TITLE
Port 'dcos marathon debug summary' in Go.

### PR DESCRIPTION
**Pull request by armandgrillet**
Resolves https://jira.mesosphere.com/browse/DCOS-60395

### CODE REVIEW

File: pkg/cmd/marathon/debug/debug_summary.go [hunk 1]
```diff
@@ -1,23 +1,130 @@
...
 		Use:   "summary",
 		Short: "Display summarized information for a queued instance launch for debugging purpose.",
 		RunE: func(cmd *cobra.Command, args []string) error {
-			return python.InvokePythonCLI(ctx)
+			client, err := marathon.NewClient(ctx)
+			if err != nil {
+				return err
+			}
+
+			return marathonDebugSummary(ctx, client, args[0], jsonOutput)
```
Comment (janisz) [inline pkg/cmd/marathon/debug/debug_summary.go]: I think we miss `Args:  cobra.ExactArgs(1),` since `args[0]` might cause panic.

Comment (janisz) [inline pkg/cmd/marathon/debug/debug_summary.go:28]: I think we miss `Args:  cobra.ExactArgs(1),`. `args[0]` can panic.

**Review by janisz — CHANGES_REQUESTED** [review]

**Review by janisz — CHANGES_REQUESTED** [review]
Linter is failing

File: pkg/cmd/marathon/debug/debug_summary.go [hunk 2]
```diff
@@ -1,23 +1,181 @@
...
+				}
+			}
+
+			table := cli.NewTable(ctx.Out(), []string{"RESOURCE", "REQUESTED", "MATCHED", "PERCENTAGE"})
+
+			if item.Application != nil {
+				table.Append([]string{
+					"ROLE",
+					nilPrintString(item.Application.Role),
+					fmt.Sprintf("%d/%d", reasons["UnfulfilledRole"], agents),
```
Comment (janisz) [inline pkg/cmd/marathon/debug/debug_summary.go]: How about extracting this to const in marathon client. The same string is used below and in another command https://github.com/dcos/dcos-core-cli/pull/421/files#diff-4522a69b4736665414222f529db51411R65-R71

**Review by janisz — CHANGES_REQUESTED** [review]

**Review by br-lewis — APPROVED** [review]

**Pull request merged**